apps/reward-manager/project.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "reward-manager",
```

Comment:
Can we rename this package to `note-reward-manager` or something for clarity? There should be an command like `yarn nx rename` that will make all the changes for you.